test/apiv2/30-volumes.at

```diff
@@ -86,6 +86,9 @@ t DELETE libpod/volumes/foo1 404 \
     .message~.* \
     .response=404
 
+# Prune volumes - bad filter input
+t POST volumes/prune?filters='garb1age}' 500
+
```

Comment:
Friendly suggestion:
```diff
-t POST volumes/prune?filters='garb1age}' 500
+t POST volumes/prune?filters='garb1age}' 500 \
+  .cause="invalid character 'g' looking for beginning of value"
```
It is a good habit when checking for failures to confirm that the failure is happening for the reason you expect, and not, e.g., for a syntax error or other unrelated reason.

Reply:
Applied friendly suggestion and rebased. Thanks!